packages/@sanity/default-login/src/LoginDialog.js

```diff
@@ -31,11 +31,18 @@ const GoogleLogo = () => (
 export default class LoginDialog extends React.Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
-    description: PropTypes.string.isRequired,
-    sanityLogo: PropTypes.node.isRequired,
-    showSanityLogo: PropTypes.bool.isRequired
+    description: PropTypes.string,
+    sanityLogo: PropTypes.node,
+    showSanityLogo: PropTypes.bool.isRequired,
```

Comment:
Why do we need both `showSanityLogo` and `sanityLogo`? Can't we just show sanity logo if its defined?

Reply:
Because sanityLogo can override the default, and showSanityLogo can toggle it.